File: 34-find-first-and-last-position-of-element-in-sorted-array/34-find-first-and-last-position-of-element-in-sorted-array.cpp
class Solution {
public:
    vector<int> searchRange(vector<int>& nums, int target) {
        int fIndex = -1;
        bool first = false;
        int sIndex = -1;
        for(int i = 0; i < nums.size(); i++){
            bool found = nums[i] == target;
            // cout << nums[i] << "  " << fIndex << "  " << sIndex << "  " <<  first << "  " << endl; 
            if(!first && found){
                fIndex = i;
                first = true;
            }
            
            if(first && found){
                sIndex = i;
            }
        
            
        }
        vector<int> res({fIndex, sIndex});
        return res;
        
    }
};

File: 34-find-first-and-last-position-of-element-in-sorted-array/34-find-first-and-last-position-of-element-in-sorted-array.py
class Solution:
    def searchRange(self, nums: List[int], target: int) -> List[int]:
        fIndex, first = -1, False
        sIndex = -1
        for i in range(len(nums)):
            found = nums[i] == target
            if not first and found:
                fIndex = i
                first = True
            
            if first and found:
                sIndex = i
        
        return [fIndex, sIndex]
                
                